examples/gazebo_scene_viewer/SceneManager.cc

```diff
@@ -1812,8 +1815,8 @@ CurrentSceneManager::~CurrentSceneManager()
 void CurrentSceneManager::OnPoseUpdate(::ConstPosesStampedPtr &_posesMsg)
 {
   // record pose timestamp
-  this->timePosesReceived = math::secNsecToDuration(
-      _posesMsg->time().sec(),  _posesMsg->time().nsec());
+  this->timePosesReceived = std::chrono::seconds(_posesMsg->time().sec()) +
+      std::chrono::nanoseconds(_posesMsg->time().nsec());
```

Comment:
had an issue with compiling against gazebo9 as it always find the math4 headers instead of math6. There could be a way to tweak cmake include directory search order but I figured it's easier to change the code so that the example works with either ign-math versions.